test/create3/Create3Factory.t.sol

```diff
@@ -118,8 +121,8 @@ contract Create3FactoryTest is DeploymentHelper, TestSetup {
         vm.revertTo(snapShot);
 
         // Second deployment set (reverse order)
-        deployChild(address(proxy), wallet, strippedBytecodeChildB);
-        deployChild(address(proxy), wallet, strippedBytecodeChildA);
+        deployChild(address(proxy), wallet, strippedBytecodeChildB, "");
```

Comment:
Shouldn't this be "0x"?

Reply:
According to [this](https://ethereum.stackexchange.com/questions/6618/in-solidity-how-can-you-set-a-bytes-variable-to-empty), no.